elliott/elliottlib/errata.py

```diff
@@ -855,3 +817,12 @@ def remove_dependent_advisories(advisory_id):
         if response.status_code != requests.codes.created:
             raise IOError(f'Failed to remove dependent {dependent} from {advisory_id}'
                           f'with code {response.status_code} and error: {response.text}')
+
+
+def get_file_meta(advisory_id):
```

Comment:
some kind of comment about what this is (e.g. link) and why it's required would be nice here

Reply:
Updated